### DISCUSSION TITLE
AzureMonitor: Resource Picker for Metrics

### DISCUSSION THREAD

**Discussion by joshhunt** [grafana/grafana#43289] [Data Sources]
- Add the Resource Picker to Metrics
 - Migrate existing subscription / resource group / namespace / resource name fields to new resourceUri field

**Comment by vickyyyyyyy** [comment]
to be created into an epic and broken down

**Comment by andresmgot** [comment]
This is now completed and will be available in Grafana 9.0. More details at https://github.com/grafana/grafana/issues/44439
